content/oci/oci.go

```diff
@@ -375,6 +376,71 @@ func (s *Store) writeIndexFile() error {
 	return os.WriteFile(s.indexPath, indexJSON, 0666)
 }
 
+// Referrers lists the descriptors of image or artifact manifests directly
+// referencing the given manifest descriptor.
+//
+// fn is called on the referrer results. If artifactType is not empty, only
+// referrers of the same artifact type are fed to fn.
+//
+// Reference: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#listing-referrers
+func (s *Store) Referrers(ctx context.Context, desc ocispec.Descriptor, artifactType string, fn func(referrers []ocispec.Descriptor) error) error {
+	var results []ocispec.Descriptor
+	predecessors, err := s.Predecessors(ctx, desc)
+	if err != nil {
+		return err
+	}
+	for _, node := range predecessors {
+		var nodeArtifactType string
```

Comment:
Annotations are missing, is it on-purpose?

Reply:
added back annotations.